spec/0.0.2/examples/template/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM goern/atomicapp-centos7 
```

Comment:
well, I just see vpavlin's and mine... no projectatomic content on hub.docker.io :/